test/test_integration_ssl.rb

```diff
@@ -189,6 +188,26 @@ def test_ssl_run_with_localhost_authority
     end
   end
 
+  def test_ssl_run_without_any_key_and_authority
+    skip_if :jruby
+    require 'logger'
+    require 'puma/minissl/context_builder'
```

Comment:
We can remove this require because if these tests run, this has been required 

https://github.com/puma/puma/blob/e3d5794a7ebe47577ced4d4dfdd6a6cc969ded01/lib/puma/binder.rb#L12-L15

https://github.com/puma/puma/blob/e3d5794a7ebe47577ced4d4dfdd6a6cc969ded01/test/test_integration_ssl.rb#L215

Reply:
When I run only this test with `bundle exec m test/test_integration_ssl.rb:195` it will fail if I don't require this file

Reply:
I see

Reply:
JFYI, test files often have duplicate requires, etc.  These are added when the test won't run as a single file.

Also, I didn't mention it, nor have I had time to look at it, but the above test doesn't need to be in `test_integration_ssl.rb`, which should just have tests that need to start a Puma process via `IO.popen`.  The above is just checking the `MiniSSL::ContextBuilder` to see if it logs missing items.  I.think.

Reply:
Good point. I will relocate the test somewhere else. I will think about this tomorrow